ci-operator/step-registry/openshift-extended/test/longduration/openshift-extended-test-longduration-commands.sh

```
@@ -265,6 +265,42 @@ if [[ $IS_ACTIVE_CLUSTER_OPENSHIFT != "false" ]]; then
     ocpVersion=$(oc get clusterversion -o json | jq -r '.items[0].status.desired.version')
 fi
 
+# Debug wait with timeout (default 3 hours) - placed AFTER cluster is ready
+DEBUG_WAIT_TIMEOUT="${DEBUG_WAIT_TIMEOUT:-10800}"
+echo "=========================================="
+echo "DEBUG WAIT ACTIVE - Cluster ready for debugging"
+echo "=========================================="
+echo "Cluster is fully provisioned and ready!"
+echo "Timeout: ${DEBUG_WAIT_TIMEOUT} seconds ($(($DEBUG_WAIT_TIMEOUT / 3600)) hours)"
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)